### ISSUE TITLE
Exercism v3

### ISSUE THREAD

**Issue by Steffan153** [joshgoebel/exercism_plus#19]
Is there a plan to make a v3 version of this? Would be kind of cool.

**Comment by joshgoebel** [comment]
Yes, but it hasn't been a priority.  I've honestly been pretty happy with v3 so far.  When I have time I'll push a commit that makes the plugin itself work with v3 (so far all it does it make the header darker and add an extension footer), but most of the functionality will need to be reconsidered / rewritten.

It's technically both harder and easier now because of all the React / API stuff... doing a "simple" plugin that hooks page loads and modifies the pages is harder now that it's all dynamic.  I'd have to play around and figure out the best way to capture page loads (and know when they complete, etc - so you know when to modify the page).

**Comment by iHiD** [comment]
> It's technically both harder and easier now because of all the React / API stuff... doing a "simple" plugin that hooks page loads and modifies the pages is harder now that it's all dynamic. I'd have to play around and figure out the best way to capture page loads (and know when they complete, etc - so you know when to modify the page).

I can help with this if you ever need it. We have hooks in the website that you could attach to I think.
